test-d/formio/components/radio.test-d.ts

```diff
@@ -0,0 +1,200 @@
+import {expectAssignable, expectNotAssignable} from 'tsd';
+
+import {RadioComponentSchema} from '../../../lib';
+
+// minimal component schema, manual:
+expectAssignable<RadioComponentSchema>({
+  id: 'yejak',
+  type: 'radio',
+  key: 'aRadio',
+  label: 'A radio',
+  openForms: {
+    dataSrc: 'manual',
+    translations: {},
+  },
+  values: [
+    {
+      value: 'dummy',
+      label: 'dummy',
+    }
+  ],
+});
+
+// minimal component schema, variable:
+expectAssignable<RadioComponentSchema>({
+  id: 'yejak',
+  type: 'radio',
+  key: 'aRadio',
+  label: 'A radio',
+  openForms: {
+    dataSrc: 'variable',
+    itemsExpression: 'dummy',
+    translations: {},
+  },
+});
+
+// multiple false and appropriate default value type
+expectAssignable<RadioComponentSchema>({
+  id: 'yejak',
+  type: 'radio',
+  key: 'aRadio',
+  label: 'A radio',
+  multiple: false,
+  defaultValue: 'dummy',
+  openForms: {
+    dataSrc: 'variable',
+    itemsExpression: 'dummy',
+    translations: {},
+  },
+});
+
+// multiple false and default value type to null
+expectAssignable<RadioComponentSchema>({
+  id: 'yejak',
+  type: 'radio',
+  key: 'aRadio',
+  label: 'A radio',
+  multiple: false,
+  defaultValue: null,
+  openForms: {
+    dataSrc: 'variable',
+    itemsExpression: 'dummy',
+    translations: {},
+  },
+});
+
+// values translations
+expectAssignable<RadioComponentSchema>({
+  id: 'yejak',
+  type: 'radio',
+  key: 'aRadio',
+  label: 'A radio',
+  multiple: false,
+  defaultValue: 'dummy',
+  openForms: {
+    dataSrc: 'manual',
+    translations: {},
+  },
+  values: [
+    {
+      value: 'dummy',
+      label: 'dummy',
+      openForms: {
+        translations: {
+          en: {
+            dummy: 'dummy_en',
```

Comment:
```suggestion
            label: 'dummy_en',
```
needs to be the label key, it's the only thing we actually translate